src/Microsoft.PowerShell.Security/security/CertificateCommands.cs

```diff
@@ -64,6 +64,18 @@ public string[] LiteralPath
         }
         private bool _isLiteralPath = false;
 
+        /// <summary>
+        /// Gets or sets the password for unlocking the certificate.
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        public SecureString Password { get; set; }
+
+        /// <summary>
+        /// Do not prompt for password if not given.
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        public SwitchParameter Force { get; set; }
```

Comment:
I'm not personally a fan of the overloaded `-Force` being the `make it happen` switch and prefer something more descriptive and specific like: `-NoPasswordPrompt`.  Let me bring it up at the next @PowerShell/powershell-committee meeting.